src/getHumanAge.test.js

```diff
@@ -7,4 +7,49 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return [0, 0] if both pets are less than 15 year old', () => {
+    expect(getHumanAge(14, 14)).toEqual([0, 0]);
+  });
+
+  test('should return [0, 0] if both pets are 0 years old', () => {
+    expect(getHumanAge(0, 0)).toEqual([0, 0]);
+  });
+
+  test(`should return [1, 1] if both pets are 15 years old
+    and less then 24`, () => {
+    expect(getHumanAge(15, 23)).toEqual([1, 1]);
+  });
+
+  test('should return [2, 2] if both pets are exactly 24 years old', () => {
+    expect(getHumanAge(24, 24)).toEqual([2, 2]);
+  });
+
+  test(`should return [2, 2] if both pets are
+    between 24 and 27 years old`, () => {
+    expect(getHumanAge(24, 27)).toEqual([2, 2]);
+  });
+
+  test('should return [3, 2] if both pets are exactly 28 years old', () => {
```

Comment:
The description of this test case seems to be incorrect. If both pets are exactly 28 years old, then according to the function's logic, they should be both considered as 3 years old in human years. Therefore, the expected result should be [3, 3], not [3, 2].